src/main/groovy/com/avioconsulting/mule/linter/rule/property/PropertyFileNamingRule.groovy

```diff
@@ -23,10 +48,18 @@ class PropertyFileNamingRule extends Rule {
 
         List propertyFilenames = []
         app.propertyFiles.each {
```

Comment:
Look at the .collect() groovy method for a simpler syntax

Reply:
tried collect(), liked it... but code narc said no... changed to: 
List propertyFilenames = app.propertyFiles*.getName()